src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -3073,7 +3073,12 @@ Bailout:
                         defaultValueDiagnostics.Clear()
                     End If
 
-                    defaultArgument = binder.GetArgumentForParameterDefaultValue(param, If(argument, methodOrPropertyGroup).Syntax, defaultValueDiagnostics, callerInfoOpt)
+                    Dim receiverOpt As BoundExpression = Nothing
+                    If candidateSymbol.IsReducedExtensionMethod() Then
+                        receiverOpt = methodOrPropertyGroup.ReceiverOpt
```

Comment:
>methodOrPropertyGroup.ReceiverOpt [](http://example.com/codeflow?start=38&length=33)

Perhaps we should assert that this value is not null (IsNot Nothing) if we get here, but the better place for the assert is on entry to this function so that we exercise it for all tested scenarios.